.gitignore

```diff
@@ -146,8 +146,9 @@ build*/
 install*/
 
 # Ignore the following bundle repositories
-!/bundle/CMakeLists.txt
-/bundle/*
+bundle/*
+!bundle/CMakeLists.txt
+!bundle/rrfs-test-data
```

Comment:
Thanks, @TingLei-NOAA 

1. I think `/bundle/*` and `bundle/*` have the same effect for RDASApp. Do we have other bundle/ directories?
But I can add `/` back. I accidentally dropped them when I debugged the issue that I could not track `bundle/rrfs-test-data` as to be discussed next

2. `bundle/*` should go first otherwise we cannot track `bundle/rrfs-test-data`. 

3. now rrfs-test-data only contains links to the real test data. So it is not a problem to track it.  
In the future, we may remove the whole rrfs-test-data/ directory. But let's leave that effort to a potential future PR.

Reply:
@guoqing-noaa 
1)  If only considering the current RDASapp, and "/bundle/*" and "bundle/*" have the same effect, we don't need to change it .   On the other hand, now, what we want to deal with is the bundle dir in the current location and it is better to let "/bundle/*" to do it exactly to avoid future impacts when other developers have bundle dir in his/her work and it will be difficult for the later to know his/her bundle dir has been labelled. in this way. 
2) , seems what I know about the order impact is different from what you stated. I will try to make an test on this. 
3) first, as I mentioned, at least for mpas-jedi data, they can't be links.  second, I prefer the minimal change principle, if it is maybe useful for future work, I would suggest leave this change to the future when it is really needed, while I am not sure if tracking links is a "convenient" way .

Reply:
@TingLei-NOAA I added back the "/" per your suggestion.

FYI,  the order in .gitignore matters, the latter ones will overwrite the previous ones.